crates/driver/src/tests/cases/protocol_fees.rs

```diff
@@ -131,40 +181,60 @@ async fn surplus_protocol_fee_buy_order_capped() {
 #[tokio::test]
 #[ignore]
 async fn surplus_protocol_fee_sell_order_capped() {
-    let fee_policy = FeePolicy::Surplus {
+    let fee_policy = Policy::Surplus {
         factor: 0.5,
-        // low enough so we get capped by volume fee
+        // high enough so we don't get capped by volume fee
         max_volume_factor: 0.1,
     };
     let test_case = TestCase {
-        order_side: order::Side::Sell,
         fee_policy,
-        order_sell_amount: 50.ether().into_wei(),
-        solver_fee: Some(10.ether().into_wei()),
-        quote_sell_amount: 50.ether().into_wei(),
-        quote_buy_amount: 40.ether().into_wei(),
-        executed: 40.ether().into_wei(),
-        executed_sell_amount: 50.ether().into_wei(),
-        executed_buy_amount: 36.ether().into_wei(),
+        order: Order {
+            amounts: Amounts {
+                sell: 50.ether().into_wei(),
+                buy: 40.ether().into_wei(),
+            },
+            side: order::Side::Sell,
+        },
+        execution: Execution {
+            // 20 ETH surplus, half of which could be captured by the settlement contract, but it is
```

Comment:
Can we align on whether the part of the surplus is captured by the driver, settlement contract or protocol (and not use a different wording in each test). I think since this is a protocol fee it should be kept by the protocol.